allocation_manager/service/unit_test.go

```diff
@@ -142,4 +142,23 @@ func TestForget(t *testing.T) {
 	assert.Equal(t, 0, u.Forget("idX"), "u.Forget second try")
 }
 
+func TestNew(t *testing.T) {
```

Comment:
Looks like this test is a work-in-progress?

Reply:
Yes, I have to figure out how to unit tests Prometheus metrics

Reply:
I think we can trust that Prometheus publishes metrics. just checking that the counters are incremented as expected should be good enough. i.e. testing our business logic (that the counter incrementing is in the right place) should be good enough.

Reply:
Do you want me to remove this test? Does it seem useless?

Reply:
removed the test:
```
allocation_manager/service/unit_test.go:175:31: metric.Get undefined (type prometheus.Counter has no field or method Get)
```